.travis.yml

```diff
@@ -87,6 +91,8 @@ after_failure:
 # check resolution...
 - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then  system_profiler SPDisplaysDataType | grep Resolution
   fi
+- if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then where python;
```

Comment:
@shriram2301 we dont need this now (was there to debug `python` install)

Reply:
@joshuef  I will remove it and commit it.